docs/src/main/asciidoc/grpc-service-implementation.adoc

```diff
@@ -83,6 +83,21 @@ public class ReactiveHelloService extends MutinyGreeterGrpc.GreeterImplBase {
 }
 ----
 
+== Blocking service implementation
+
+By default, all the methods from a gRPC service run on the event loop.
+As a consequence, you must **not** block.
+If your service logic must bock, annotate the method with `io.smallrye.common.annotation.Blocking`:
+
```

Comment:
Hi @cescoffier, just noticed that you have a typo here. It says "bock" instead of "block"

Reply:
Thanks! I fixed it.